app/bundles/EmailBundle/Controller/EmailController.php

```diff
@@ -1304,7 +1304,7 @@ function (&$field) {
         ];
 
         // Send to current user
-        $errors = $model->sendEmailToUser($entity, $users, $fields, [], [], false);
+        $errors = $model->sendSampleEmailToUser($entity, $users, $fields, [], [], false);
```

Comment:
The third attribute here is `$fields`, but in the method definition, it should be a `$lead`. Is this correct?